website/components/TableSearch.tsx

```diff
@@ -0,0 +1,74 @@
+import "instantsearch.css/themes/reset.css";
+import React from "react";
+import Link from "next/link";
+import algoliasearch from "algoliasearch/lite";
+import {
+  InstantSearch,
+  connectSearchBox,
+  connectStateResults,
+} from "react-instantsearch-dom";
+
+const searchClient = algoliasearch(
+  "0OUG5EUZ6H",
+  "fa5e5811f7275c275157c74b46f11345"
+);
+
+const SearchBox = ({ refine }) => {
```

Comment:
This needs CSS, we can probably reuse https://github.com/shuding/nextra/blob/32cd3855cd0ae3c7687423bae6308a70cc28fd4c/packages/nextra-theme-docs/src/components/search.tsx